src/main/java/com/fundation/logic/view/PlayerFrame.java

```diff
@@ -72,28 +75,23 @@ private void initComponent(String path) {
         volumeSlider = new JSlider(JSlider.VERTICAL, 0, 100, volumeLevel);
         player = new EmbeddedMediaPlayerComponent();
         fileToBePlayed = new File(path);
-        buttonListener = new ButtonListener(volumeLevel);
+        buttonListener = new PlayerButtonListener(volumeLevel);
     }
 
     /**
-     * Set all PlayerJFrame components
+     * Set all PlayerFrame components
      */
     private void initSetting() {
         setBounds(new Rectangle(80, 100, 800, 540));
```

Comment:
In this line you have use magical number.

Reply:
Magical numbers changed to constants